canonical/ubuntu.com
# WD-2123 & WD-2125 - Create MongoDB product page & contact form

## Done

- Create the MongoDB product page
- Create the MongoDB contact form
- Drivebys

## QA

- Check out the [demo](https://ubuntu-com-12769.demos.haus/data/mongodb) and compare it with [the design](https://www.figma.com/file/XAmHDBtr7cjQOeDrsE4TH7/23.04-u.com%2Fdata%3A-MongoDB-%26-OpenSearch?node-id=132-2316&t=wt8d2LQ4UdWxyWGj-4) and [copy docs](https://docs.google.com/document/d/1_JuxbWsqTeQRRmSL0sCpM3alabr06WX4Cb-tt1BBW_I/edit)

## Issue / Card

- [WD-2123](https://warthogs.atlassian.net/browse/WD-2123)
- [WD-2125](https://warthogs.atlassian.net/browse/WD-2125)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-2123]: https://warthogs.atlassian.net/browse/WD-2123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WD-2125]: https://warthogs.atlassian.net/browse/WD-2125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
